apis/Google.Bigquery.V2/Google.Bigquery.V2.Snippets/BigqueryClientSnippets.cs

```diff
@@ -637,28 +638,622 @@ public void ParameterizedQuery_PositionalParameters()
             var resultsList = client.ExecuteQuery(command)
                 .PollUntilCompleted()
                 .GetRows()
-                .Select(row => new { Name = (string)row["player"], Score = (long)row["score"], Level = (long)row["level"] })
+                .Select(row => new { Name = (string) row["player"], Score = (long) row["score"], Level = (long) row["level"] })
                 .ToList();
-            
+
             Assert.Contains(new { Name = "Tim", Score = 5310L, Level = 3L }, resultsList);
             Assert.Contains(new { Name = "Tim", Score = 2000L, Level = 2L }, resultsList);
             Assert.Contains(new { Name = "Nadia", Score = 8310L, Level = 5L }, resultsList);
             Assert.DoesNotContain(new { Name = "Tim", Score = 503L, Level = 1L }, resultsList);
             Assert.DoesNotContain(new { Name = "Nadia", Score = 1320L, Level = 2L }, resultsList);
         }
 
+        [Fact]
+        public async Task QueryOverviewAsync()
+        {
+            string projectId = _fixture.ProjectId;
+
+            // Sample: QueryOverview
+            BigqueryClient client = await BigqueryClient.CreateAsync(projectId);
+            BigqueryTable table = await client.GetTableAsync("bigquery-public-data", "samples", "shakespeare");
+
+            string sql = $"SELECT corpus AS title, COUNT(word) AS unique_words FROM {table} GROUP BY title ORDER BY unique_words DESC LIMIT 10";
+            BigqueryQueryJob query = await client.ExecuteQueryAsync(sql);
+            query = await query.PollUntilCompletedAsync();
+
+            await query.GetRowsAsync().ForEachAsync(row =>
+            {
+                Console.WriteLine($"{row["title"]}: {row["unique_words"]}");
+            });
+            // End sample
+        }
+
+        [Fact]
+        public async Task LegacySqlOverviewAsync()
+        {
+            string projectId = _fixture.ProjectId;
+
+            // Sample: LegacySql
+            BigqueryClient client = await BigqueryClient.CreateAsync(projectId);
+            BigqueryTable table = await client.GetTableAsync("bigquery-public-data", "samples", "shakespeare");
+
+            string sql = $"SELECT TOP(corpus, 10) AS title, COUNT(*) AS unique_words FROM {table:legacy}";
+            BigqueryQueryJob query = await client.ExecuteQueryAsync(sql, new ExecuteQueryOptions { UseLegacySql = true });
+            query = await query.PollUntilCompletedAsync();
+
+            await query.GetRowsAsync().ForEachAsync(row =>
+            {
+                Console.WriteLine($"{row["title"]}: {row["unique_words"]}");
+            });
+            // End sample
```

Comment:
We could do - but the main purpose of these snippets is for documentation purposes. Will consider that as a general purpose extra way of testing.